tests/parsers/moldev_softmax_pro/testdata/MD_SMP_absorbance_endpoint_example04.json

```diff
@@ -27,10 +27,6 @@
                             "absorbance": {
                                 "value": 1.328,
                                 "unit": "mAU"
-                            },
-                            "compartment temperature": {
-                                "value": 0.0,
-                                "unit": "degC"
```

Comment:
Note to reviewer: #635 was supposed to remove all `compartment temperature`s with a value of `0`. However, it only removed them from `fluorescence` measurements (oops)